tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbSqlStorageTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Bot.Builder.Core.Extensions;
+using Microsoft.Bot.Builder.Core.Extensions.Tests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.Builder.Azure.Tests
+{
+    [TestClass]
+    [TestCategory("Storage")]
+    [TestCategory("Storage - CosmosDB SQL")]
+    public class CosmosDbSqlStorageTests : StorageBaseTests
+    {
+        private const string CosmosServiceEndpoint = "https://localhost:8081";
+        private const string CosmosAuthKey = "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
```

Comment:
Can't have this in GitHub. Please follow the same pattern we're using in the other tests. 

This key, as it was uploaded to a public repo, needs to be depricated (unless it's only a local-key or similar).

Reply:
It's the endpoint & authkey for the CosmosDB emulator, similar to the Azure Storage emulator. It is a well-known key. I'll add a comment about it.